cv32e40p/README.md

```diff
@@ -14,3 +14,23 @@ There are README files in each directory with additional information.
 
 ## Getting Started
 Check out the Quick Start Guide in the [CORE-V-VERIF Verification Strategy](https://core-v-docs-verif-strat.readthedocs.io/en/latest/).
+
+## Setup
+
+```bash
+export CV_SW_PREFIX=riscv32-unknown-elf-
```

Comment:
This works for your environment, but will not work for everyone.   There are already detailed instructions for this in the `mk` directory's [README](https://github.com/openhwgroup/core-v-verif/blob/cv32e40p/release/mk/README.md) and [TOOLCHAIN](https://github.com/openhwgroup/core-v-verif/blob/cv32e40p/release/mk/TOOLCHAIN.md) docs.

Reply:
Please ignore this commit.